graphql_api/types/repository/repository.py

```diff
@@ -587,13 +616,24 @@ def resolve_coverage_analytics(
 async def resolve_test_results_aggregates(
     repository: Repository,
     info: GraphQLResolveInfo,
+    history: MeasurementInterval | None = None,
+    **_,
 ):
+    history = convert_history_to_timedelta(history)
     return await sync_to_async(generate_test_results_aggregates)(
-        repoid=repository.repoid
+        repoid=repository.repoid, history=history
     )
 
 
 @repository_bindable.field("flakeAggregates")
 @convert_kwargs_to_snake_case
```

Comment:
we actually don't need the @convert_kwargs decorator anymore due to https://github.com/codecov/codecov-api/blob/9aa6458ad023fe0ec55993ad9134bbd0af5fb3f7/graphql_api/schema.py#L7